recap/clients/postgresql.py

```diff
@@ -100,6 +101,16 @@ def schema(self, catalog: str, schema: str, table: str) -> StructType:
                 JOIN pg_catalog.pg_attribute
                     ON pg_catalog.pg_attribute.attrelid = pg_catalog.pg_class.oid
                     AND pg_catalog.pg_attribute.attname = information_schema.columns.column_name
+                LEFT JOIN pg_catalog.pg_type
```

Comment:
Minor nit, but can you add two `--` comments in this query to break denote the part of the query that's for the array dimensions and the part that's for enums? It's just starting to get complex enough that some additional docs would be helpful.